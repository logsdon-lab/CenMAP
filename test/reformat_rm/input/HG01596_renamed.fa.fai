version https://git-lfs.github.com/spec/v1
oid sha256:9fb2b1ead305be15d891f4f544d6bd4c9a6b2a93082ddbbd2107bf850716de53
size 257
