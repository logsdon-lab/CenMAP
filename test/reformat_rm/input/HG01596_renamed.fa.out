version https://git-lfs.github.com/spec/v1
oid sha256:2d5978cc760a7fb2373e87ae707082bdf055d85cb5076a32c3b9297a89af315e
size 2050886
